### DISCUSSION TITLE
Discussions Topics for Dec 20, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#3540] [Show and tell]
Discussions Topics for Dec 20, 2021 Engineering Sync

**Comment by jimduff-usds** [comment]
I'd like to show https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/playbooks/how-to-use-filters.md
I'll demo how to "fix" an existing yamlito.
As part of this, I can demo @RickHawesUSDS 's new yamlito "diff" tool as well.
